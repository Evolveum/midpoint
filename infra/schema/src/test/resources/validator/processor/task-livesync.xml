<!--
  ~ Copyright (C) 2010-2023 Evolveum and contributors
  ~
  ~ This work is dual-licensed under the Apache License 2.0
  ~ and European Union Public License. See LICENSE file for details.
  -->

<task xmlns="http://midpoint.evolveum.com/xml/ns/public/common/common-3"
        xmlns:mext="http://midpoint.evolveum.com/xml/ns/public/model/extension-3"
        xmlns:rext="http://midpoint.evolveum.com/xml/ns/public/report/extension-3"
        oid="979ec101-d36c-49ea-a6c1-5d0c087d9689">

    <name>livesync</name>
    <extension>
        <rext:reportOutputOid>e7df0b0e-48f7-4c32-a4a5-c444a48d08f0</rext:reportOutputOid>
        <mext:liveSyncErrorHandlingStrategy>
            <entry id="1">
                <order>1</order>
                <situation>
                    <status>fatal_error</status>
                </situation>
            </entry>
        </mext:liveSyncErrorHandlingStrategy>
    </extension>
    <expectedTotal>123</expectedTotal>
    <activity>
        <work>
            <cleanup>
                <policies>
                    <objectResults>
                        <maxAge>P1D</maxAge>
                        <maxRecords>10</maxRecords>
                    </objectResults>
                </policies>
            </cleanup>
        </work>
    </activity>
</task>
