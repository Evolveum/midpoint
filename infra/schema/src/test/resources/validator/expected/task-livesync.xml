<!--
  ~ Copyright (C) 2010-2023 Evolveum and contributors
  ~
  ~ This work is dual-licensed under the Apache License 2.0
  ~ and European Union Public License. See LICENSE file for details.
  -->

<task xmlns="http://midpoint.evolveum.com/xml/ns/public/common/common-3"
        xmlns:mext="http://midpoint.evolveum.com/xml/ns/public/model/extension-3"
        xmlns:rext="http://midpoint.evolveum.com/xml/ns/public/report/extension-3"
        oid="979ec101-d36c-49ea-a6c1-5d0c087d9689">

    <name>livesync</name>
    <extension>
        <rext:reportDataParam oid="e7df0b0e-48f7-4c32-a4a5-c444a48d08f0"/>
    </extension>
    <expectedTotal>123</expectedTotal>
    <activity>
        <work>
            <cleanup>
                <policies>
                    <simpleOperationExecutions>
                        <maxAge>P1D</maxAge>
                        <maxRecords>10</maxRecords>
                    </simpleOperationExecutions>
                    <complexOperationExecutions>
                        <maxAge>P1D</maxAge>
                        <maxRecords>10</maxRecords>
                    </complexOperationExecutions>
                </policies>
            </cleanup>
        </work>
        <controlFlow>
            <errorHandling>
                <entry id="1">
                    <order>1</order>
                    <situation>
                        <status>fatal_error</status>
                    </situation>
                </entry>
            </errorHandling>
        </controlFlow>
    </activity>
</task>
